Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -766,12 +763,7 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
         private System.Numerics.Matrix4x4 GetSceneToWorldTransform()
         {
             var result = System.Numerics.Matrix4x4.Identity;
-
-            if (Application.isEditor)
-            {
-                return result;
-            }
-
+#if WINDOWS_UWP
```

Comment:
Going forward, we could also choose to re-add this code using package-specific `#defines`, like we added in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9317/files. That way, we'd only try to run this code if DotNetWinRT is installed, instead of assuming it will be.